pkg/translate/field_mask.go

```diff
@@ -0,0 +1,24 @@
+package translate
+
+import (
+	"fmt"
+
+	"go.expect.digital/translate/pkg/model"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/fieldmaskpb"
+)
+
+// parseFieldMask parses and normalizes a field mask from a proto message and a list of paths.
+func parseFieldMask(message proto.Message, paths []string) (model.FieldMask, error) {
```

Comment:
Future-proofing, when we will have multiple entities with field mask we won't be needing to call `fieldmaskpb.New` and `fieldmaskpb.Normalize` all over again